CMakeLists.txt

```diff
@@ -163,9 +163,12 @@ endif()
 
 set_target_properties(${STLINK_LIB_STATIC} PROPERTIES OUTPUT_NAME ${PROJECT_NAME})
 
-install(TARGETS ${STLINK_LIB_STATIC}
-	ARCHIVE DESTINATION ${STLINK_LIBRARY_PATH}
-)
+set(STATIC_LIB_INSTALL ON CACHE BOOL "Install static lib")
```

Comment:
Place the flag at the top of the cmake file so people know they can control the build. Rename the flag to `STLINK_STATIC_LIB`.